docs/logs/discovery/tailing-log-files.md

```diff
@@ -0,0 +1,54 @@
+title: Shipping Log Files
+description: Shipping Log File Sources
+
+Enabling automatic logs shipping is accomplished by triggering the "Set up" button that opens the flyout for configuring log file shipping.
+
+![Setup Button](images/setup.png)
+
+In the flyout panel, you have to repeat the following steps to start shipping logs for the log file source:
+
+1. Enable the "Automatically ship <group-name> logs" toggle
+2. Select the destination Logs App. If the Logs App doesn't exist, you'll be able to create one from the same screen
+3. Click the `Save Changes` button
+
+The log shipper will automatically start tailing any log file that pertains to the service type for which the log shipping was set up.
+
+![Enable Log Shipping](images/logfile-flyout.png)
+
+## Defining glob patterns
+
+To narrow down which log files should be tailed, you can define include/exclude [glob](https://en.wikipedia.org/wiki/Glob_(programming)) patterns. The following glob wildcards are supported:
+
+- `*` matches any sequence of non-separator characters
+- `?` matches any single non-separator character
+- `[0-9]` matches character ranges
+
+When defining glob patterns, you can either specify include or exclude glob expressions. Exclude glob patterns take precedence over include patterns. This means, if there is a log file matched by both include and exclude glob patterns, it will results in the log file not being shipped.
```

Comment:
Maybe mention if include is empty everything is included

Reply:
Already mentioned this in the previous section:

`The log shipper will automatically start tailing any log file that pertains to the service type for which the log shipping was set up.`